docs/kubernetes/windows-details.md

```diff
@@ -0,0 +1,335 @@
+# More on Windows and Kubernetes
+
+If you're trying to deploy Kubernetes with Windows the first time, be sure to check out the [quick start](windows.md) first. If you're looking for more details on deployments, examples or troubleshooting read on.
+
+## Customizing Windows deployments
+
+ACS-Engine allows a lot more customizations available in the [docs](../), but here are a few important ones you should know for Windows deployments. Each of these are extra parameters you can add into the ACS-Engine apimodel file (such as kubernetes-windows.json from the quick start) before running `acs-engine generate`

```

Comment:
Maybe put backticks around the filename `kubernetes-windows.json`? Also the final sentence needs a period.

Reply:
Thanks! fixed in new changelist